modules/nw-ne-openshift-ingress.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+// * understanding-networking.adoc
+
+
+[id="nw-ne-openshift-ingress_{context}"]
+= {product-title} Ingress Operator
+When you create your {product-title} cluster, pods and services running on the cluster are each allocated their own IP addresses. The IP addresses are accessible to other pods and services running nearby but not accessible to outside clients. The Ingress Operator implements the `ingresscontroller` API and is the component responsible for enabling external access to {product-title} cluster services.
```

Comment:
* but not -> but are not ?
* Should `ingresscontroller` API be capitalized differently, like `IngressController` API ?